instrumentation/servlet/servlet-3.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/servlet/v3_0/snippet/ServletOutputStreamInjectionHelper.java

```diff
@@ -42,7 +42,12 @@ public static boolean handleWrite(
     state.setAlreadyInjected(); // set before write to avoid recursive loop
     out.write(original, off, i + 1);
     try {
-      byte[] snippetBytes = SnippetHolder.getSnippetBytes(state.getCharacterEncoding());
+      if (state.getWrapper().isNotSafeToInject()) {
+        // content length already set and sent, don't inject
```

Comment:
I don't think this comment is needed anymore with the nice method name above `isNotSafeToInject`
```suggestion
```